src/exceptions/user_denied_transaction_exception.ts

```diff
@@ -0,0 +1,7 @@
+export class UserDeniedTransactionSignatureException extends Error {
+    constructor(m: string = 'User denied transaction signature') {
```

Comment:
R: Maybe we can use the default constant `USER_DENIED_TRANSACTION_SIGNATURE_ERR` right here